### ISSUE TITLE
Cannot resolve module

### ISSUE THREAD

**Issue by knowbody** [FormidableLabs/victory#144]
Added victory to the React component and I get bunch of errors, like: `Cannot resolve module...`

```
ERROR in ./~/victory/~/victory-bar/lib/components/victory-bar.js
Module not found: Error: Cannot resolve module 'radium' in /Users/rye/scm/vendor/react-boilerplate/node_modules/victory/node_modules/victory-bar/lib/components
 @ ./~/victory/~/victory-bar/lib/components/victory-bar.js 23:14-31

… SNIPPED - MORE ERRORS …
```

To reproduce:

1. Clone the repo from: https://github.com/knowbody/react-boilerplate/tree/victory-playground
2. Check out to `victory-playground` branch
3. `npm install`
4. `npm start`

After that all of the errors shown in the above screenshot should be thrown.

Node v4.0.0
Npm v2.14.2

**Comment by ryan-roemer** [comment]
Please try with npm v3 and report back. There's a known npm v2 bug that the npm folks have deemed `wontfix`

**Comment by knowbody** [comment]
oh I forgot about that, thanks again @ryan-roemer, checking now :thought_balloon:

**Comment by ryan-roemer** [comment]
That said, I don't think that bug should manifest on victory installs from CDN anymore if we added the `cd lib || builder run npm:postinstall`

**Comment by knowbody** [comment]
yup npm v3 solves this issue, thanks!

**Comment by knowbody** [comment]
this can be closed now as we have #145 and #146 opened
